tee-worker/ts-tests/utils.ts

```diff
@@ -32,6 +32,9 @@ const crypto = require('crypto');
 // TODO add self signed certificate ??
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
 
+// maximum number of blocks that we wait in listening events before we timeout
+const listenTimeoutInBlock = 15;
```

Comment:
When I use [litentry-runtime](https://github.com/litentry/integritee-node/tree/litentry-runtime) as the parent chain, 15 blocks may not be enough, so calculate `listenTimeoutInBlock` by `slot_duration` in the original code

Reply:
I see, reverted.